src/s3a_the_gutter.tw2

```diff
@@ -1,174 +1,913 @@
+
+[[I walked into the sign’s field of light|The Gutter - Outside hub]].]]
+[>>Derek loves (fuckin) Whitnee](click:"Derek loves (fuckin) Whitnee")[(replace: ?gutterSubSign)[//Derek loves (fuckin) Whitnee//
+
+This poignant shout-out had been there forever. The (fuckin) was a later addition and no one was sure if it was from the original artist.
+
+[[I walked into the sign’s field of light|The Gutter - Outside hub]].]]]]<gutterSubSign|
+<!-- This makes the blinking neon sign effect work -->
+(set: $guttersign1 to "<span class='neon'><span class='blinking'>THE</span> GUTTER</span>")(set: $guttersign2 to "<span class='neon'>THE <span class='blinking2'>G</span>UTTER</span>")(set: $guttersign3 to "<span class='neon'>THE GUTT<span class='blinking3'>ER</span></span>")
+<style> .neon {
+  font-size: 2em;
+  font-family: copperplate;
+  color: transparent;
+	text-shadow:
+		0em    0em 0.02em rgba(255,255,255,0.75),
+		-0.2em 0em 0.5em  rgba(255,255,255,0.5),
+		0.2em  0em 0.5em  rgba(255,255,255,0.5);
+	text-decoration: none;
+}
+.blinking {
+  animation: fade-in-out .3s ease-in-out infinite alternate;
+	-webkit-animation: fade-in-out .3s ease-in-out infinite alternate;
+}
+.blinking2 {
+  animation: fade-in-out 1.2s ease-in-out infinite alternate;
+	-webkit-animation: fade-in-out 1.2s ease-in-out infinite alternate;
+}
+.blinking3 {
+  animation: fade-in-out .8s ease-in-out infinite alternate;
+	-webkit-animation: fade-in-out .8s ease-in-out infinite alternate;
+}
+</style>
+<!--These are the variables that need to be set for the Outside hub passages to work properly-->
+(set: $oldSchoolJuggalosSeen to false)(set: $jarSeen to false)(set: $justLeftTheGutter to false)(set: $golfCartSeen to false)(set: $cartPartsSeen to 0)(set: $wheelsSeen to false)(set: $engineSeen to false)(set: $defenseSeen to false)(set: $hornSeen to false)(set: $gutterStatus to "intro")(set: $gutterOutside to "hub")
+
+::The Gutter - Name prompt outside hub
+{ (set: $playerName to (prompt: "Enter name:", ""))
+(if: (text: $playerName)'s length > 0)[Right, my name is $playerName. [[Moving on|The Gutter - Outside hub]].]
+}(else:)[No, [[that's not right|The Gutter - Name prompt outside hub]].]
+
+(set: $dissolve to (transition: 'dissolve') + (transition-time: 3s))
+(set: $hatchet to true)(set: $multitool to true)
+(set: $oldSchoolJuggalosSeen to false)(set: $jarSeen to false)(set: $justLeftTheGutter to false)(set: $golfCartSeen to false)(set: $cartPartsSeen to 0)(set: $wheelsSeen to false)(set: $engineSeen to false)(set: $defenseSeen to false)(set: $hornSeen to false)(set: $gutterStatus to "intro")
+
+::The Gutter - Outside hub
+(if: $justLeftTheGutter is true)[(set: $justLeftTheGutter to false)I walked back into the dim neon light of the Gutter’s sign. I could see:]
+\(else:)[In the dim light of the neon sign, I could see:]
+
+>>[[The Gutter’s beaten-to-hell block of a front door|The Gutter - Inside $gutterStatus]]
```

Comment:
Just a note to myself, mostly, but in some places actions have a single `>`, whereas in others they have two `>>`. I think this is something we should fine-tune after we've refined the game's CSS more — depending on fonts/kerning/etc., one or the other might actually look better. Either way, we should add this to our punch list for final review.

Reply:
Created: https://github.com/whymog/efjm/issues/20

Reply:
Thanks for creating that punch list!